BoxContentSDK/BoxContentSDK/Models/BOXRepresentation.m

```diff
@@ -87,4 +87,36 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
     return self;
 }
 
+- (NSDictionary *)JSONDictionary
+{
+    NSMutableDictionary *JSONData = [NSMutableDictionary new];
+    JSONData[BOXAPIObjectKeyRepresentation] = self.type;
+
+    if (self.dimensions) {
+        JSONData[BOXAPIObjectKeyProperties] = @{BOXAPIObjectKeyDimensions: self.dimensions};
+    }
+
+    NSMutableDictionary *statusDict = nil;
+    if (self.status || self.statusCode) {
+        statusDict = [NSMutableDictionary new];
+        statusDict[BOXAPIObjectKeyState] = self.status;
+        statusDict[BOXAPIObjectKeyCode] = self.statusCode;
+    }
+    JSONData[BOXAPIObjectKeyStatus] = statusDict;
+
+    JSONData[BOXAPIObjectKeyDetails] = self.details;
+
+    NSString *contentURLString = self.contentURL.absoluteString;
+    if (contentURLString && [self.type isEqualToString:BOXRepresentationTypeHLS]) {
+        contentURLString = [contentURLString stringByReplacingOccurrencesOfString:BOXRepresentationTemplateValueHLSManifest withString:BOXRepresentationTemplateKeyAccessPath];
+    }
+    if (contentURLString) {
+        JSONData[BOXAPIObjectKeyContent] = @{BOXAPIObjectKeyURLTemplate: contentURLString};
+    }
+    if (self.infoURL.path) {
```

Comment:
Odd to check for path, then set absoluteString.

Reply:
updated